### PR TITLE
do: added, modified, renamed status

### PR DESCRIPTION
**Pull request by andreluizlunelli**

### CODE REVIEW

File: AddedClassRename.php [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?php
+
+class MyClass
+{
+	public function main() {}
```
Comment (nelsondevsquad) [inline AddedClassRename.php:5]: This function is empty! Don't you know that's an anti-pattern?

- Graham C

File: AddedClassRename.php [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?php
```
Comment (nelsondevsquad) [inline AddedClassRename.php:1]: PHP is a terrible language. What do you think about starting this product over using C?

- Linus T

**Review by nelsondevsquad — COMMENTED** [review]
Great code, loved it.

File: AddedClassRename.php (hunk 2 above)
Comment (samuel-dos-santos) [inline AddedClassRename.php:1]: This is wrong. I recommend the short version <?

File: AddedClassRename.php [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+class MyClass
+{
+	public function main() {}
+}
```
Comment (samuel-dos-santos) [inline AddedClassRename.php:6]: Hello! It's me, Mario!

**Review by samuel-dos-santos — APPROVED** [review]
Alguma coisa aqui

**Pull request merged**